signalfx/ingest.py

```diff
@@ -273,27 +274,63 @@ def _set_event_properties(self, pbuf_obj, properties):
 
     def _assign_property_value(self, prop, value):
         if isinstance(value, int):
+            if value > 9223372036854775807:
+                raise ValueError('Invalid Property Value ' + str(value) +
+                                 ' exceeds maximum signed 64 bit integer' +
+                                 ' (9223372036854775807)')
+            if value < -9223372036854775808:
+                raise ValueError('Invalid Property Value ' + str(value) +
+                                 ' less than minimum signed 64 bit integer' +
+                                 ' (-9223372036854775808)')
             prop.value.intValue = value
+        elif sys.version_info < (3,) and isinstance(value, long):
```

Comment:
It's bad practice to have code that is conditional on the Python version. Use `six.integer_types`.